common/locales/en/generic.json

```diff
@@ -38,6 +38,7 @@
     "alreadyHave": "Whoops! You already have this item. No need to buy it again!",
     "delete": "Delete",
     "gems": "Gems",
+    "startGems": "You have ",
```

Comment:
To make this better for translators, I'd actually like to see this be:

```
"gemButton": "You have <%= number %> Gems.",
```

[Here's an example of how to use a translatable string variable in a jade file](https://github.com/HabitRPG/habitrpg/blob/develop/website/views/shared/header/menu.jade#L198)